src/helpers/annotation-metadata.ts

```diff
@@ -0,0 +1,115 @@
+export type Annotation = {
+  uri: string;
+  document: { title: string };
+  links: {
+    /** A "bouncer" URL that takes the user to see the annotation in context */
+    incontext?: string;
+    /** URL to view the annotation by itself. */
+    html?: string;
+  };
+};
```

Comment:
I have created this type as small as possible, just with the minimum properties that are needed by the helpers defined here.

In future we plan to extract the type definitions to a shared location.